framework/db/worklist_manager.py

```diff
@@ -127,25 +116,19 @@ def add_work(self, target_list, plugin_list, force_overwrite=False):
                     # Check if it is already run ;) before adding
                     if ((force_overwrite is True) or
                         (force_overwrite is False and
-                            self.plugin_output.PluginAlreadyRun(
-                                plugin, target_id=target["id"]) is False)):
+                            self.plugin_output.PluginAlreadyRun(plugin, target_id=target["id"]) is False)):
                         # If force overwrite is true then plugin output has
                         # to be deleted first
```

Comment:
https://github.com/hTrap/owtf/blob/pep8-fixes/framework/db/worklist_manager.py#L113 - last arg in line with the first one